### PR TITLE
Add new Advanced Setting for Throughput Limits

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-33333

### Staged Changes

https://preview-mongodbmongocaleb.gatsbyjs.io/atlas-app-services/docsp-33333/triggers/database-triggers/#std-label-triggers-maximum-throughput

### CODE REVIEW

File: source/triggers/database-triggers.txt [hunk 1]
```diff
@@ -234,6 +234,20 @@ options are available:
 
      - .. include:: /includes/trigger-auto-resume.rst
 
+   * - :guilabel:`Maximum Throughput Triggers`
+
+     - .. _triggers-maximum-throughput:
+      
+       If your App Services app is running on a dedicated server (M10+ Tier),
```
Comment (nlarew) [inline source/triggers/database-triggers.txt:241]:
```suggestion
       If the linked data source is a dedicated server (M10+ Tier),
```

File: source/triggers/database-triggers.txt [hunk 2]
```diff
@@ -234,6 +234,20 @@ options are available:
 
      - .. include:: /includes/trigger-auto-resume.rst
 
+   * - :guilabel:`Maximum Throughput Triggers`
+
+     - .. _triggers-maximum-throughput:
+      
+       If your App Services app is running on a dedicated server (M10+ Tier), 
+       you can increase the maximum throughput from the default 10K to 50K.
```
Comment (nlarew) [inline source/triggers/database-triggers.txt:242]: Can we link to the [Event Processing Throughput](https://www.mongodb.com/docs/atlas/app-services/triggers/#event-processing-throughput) limitation?

Also we don't define throughput on this page - can we clarify that 10K and 50K refer to the maximum number of events that can be processed simultaneously

**Review by nlarew — CHANGES_REQUESTED** [review]
A couple changes. Also, do we have corresponding tickets for the Admin API / Config Files vesrions of this feature?

**Comment by MongoCaleb** [comment]
Created https://jira.mongodb.org/browse/DOCSP-33598 to track the API/config file changes

File: source/triggers.txt [hunk 3]
```diff
@@ -81,6 +83,11 @@ Trigger's configuration as well as the logic in the Trigger function.
 Ordered triggers process events from the queue one at a time. An
 unordered trigger can process up to 10,000 events concurrently.
 
+.. note::
+
+   If the linked data source is a dedicated server (M10+ Tier), 
+   your app can process up to 50,000 events concurrently.
```
Comment (nlarew) [inline source/triggers.txt:89]:
```suggestion
   you can configure individual triggers to process up to 50,000 events concurrently.
```

The setting is per-trigger right?

File: source/triggers/database-triggers.txt [hunk 4]
```diff
@@ -234,6 +234,21 @@ options are available:
...
+   * - :guilabel:`Maximum Throughput Triggers`
+
+     - .. _triggers-maximum-throughput:
+      
+       If the linked data source is a dedicated server (M10+ Tier), 
+       you can increase the :ref:`maximum throughput <event_processing_throughput>`
+       from the default 10,000 concurrent processes to 50,000. 
+
+       Before increasing the maximum throughput, consider whether one or more of 
+       your triggers are calling a rate-limited external API. Increasing the
```
Comment (nlarew) [inline source/triggers/database-triggers.txt:246]:
```suggestion
       Before increasing the maximum throughput, consider whether the
       trigger is calling a rate-limited external API. Increasing the 
```

The limit & setting are per-trigger so you really only need to consider the individual triggers you enable

**Review by nlarew — APPROVED** [review]
LGTM % one more suggestion

**Review by jasonrydberg — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**